includes/features/SU-SOE/stanford_jumpstart_home/homepage_hoover.feature

```diff
@@ -22,7 +22,7 @@ Feature: Hoover Homepage Layout
     | Affiliates            | Main Bottom           |
     | Contact Us            | Footer                |
 
-  @deploy  @live @safe
+  @safe
   Scenario Outline: Homepage content hoover
```

Comment:
The content in this test will change when the site is launched. The tests should be @deploy and not @safe. Please check the other tests in this file as well.